requirements/requirements.txt

```diff
@@ -16,3 +16,4 @@ ray
 sentencepiece
 google
 protobuf
+pytest
```

Comment:
This should be test requirements, right?

Reply:
Do we need to update triton's version in requirements-test.txt?

Reply:
> This should be test requirements, right?
right, will fix it.

Reply:
fix it.